app/screens/interactive_dialog/interactive_dialog.js

```diff
@@ -47,12 +47,27 @@ export default class InteractiveDialog extends PureComponent {
         this.state = {
             values,
             errors: {},
+            isLandscape: this.isLandscape(),
             submitting: false,
         };
     }
 
     componentDidMount() {
         this.navigationEventListener = Navigation.events().bindComponent(this);
+        Dimensions.addEventListener('change', this.orientationDidChange);
```

Comment:
Is the listener required? Getting the dimension in the render method, at times is all you need. Try it out, if it works (I’m 99% sure) you can remove the listener

Reply:
Yes, it's needed.  I tried first on the render method and works fine on current orientation but didn't update on orientation change.